plugins/Goals/lang/en.json

```
@@ -3,63 +3,63 @@
         "CategoryTextVisitsSummary_VisitsSummary": "User attribute",
         "CategoryTextDevicesDetection_DevicesDetection": "Devices",
         "CategoryTextGeneral_Actions": "Pages",
         "CategoryTextGeneral_Visit": "engagement",
-        "ClickOutlink": "Click on a Link to an external website",
+        "ClickOutlink": "Click on a link to an external website",
         "SendEvent": "Send an event",
-        "ColumnAverageOrderRevenueDocumentation": "Average Order Value (AOV) is the total revenue from all Ecommerce Orders divided by the number of orders.",
+        "ColumnAverageOrderRevenueDocumentation": "Average order value (AOV) is the total revenue from all e-commerce orders divided by the number of orders.",
         "ColumnAveragePriceDocumentation": "The average revenue for this %s.",
-        "ColumnAverageQuantityDocumentation": "The average quantity of this %s sold in Ecommerce orders.",
-        "ColumnConversionRateDocumentation": "The percentage of visits that triggered the goal %s.",
-        "ColumnConversionRatePageViewedBeforeDocumentation": "The percentage of all conversions for goal %s where this page was viewed before conversion.",
+        "ColumnAverageQuantityDocumentation": "The average quantity of this %s sold in e-commerce orders.",
+        "ColumnConversionRateDocumentation": "The percentage of visits that triggered the %s goal.",
+        "ColumnConversionRatePageViewedBeforeDocumentation": "The percentage of all conversions for the %s goal where this page was viewed before conversion.",
         "ColumnConversionRateProductDocumentation": "The %s conversion rate is the number of orders containing this product divided by number of visits on the product page.",
         "ColumnConversions": "Conversions",
         "Conversion": "Conversion",
         "ColumnConversionsDocumentation": "The number of conversions for %s.",
-        "ColumnOrdersDocumentation": "The total number of Ecommerce orders which contained this %s at least once.",
-        "ColumnPurchasedProductsDocumentation": "The number of purchased products is the sum of Product quantities sold in all Ecommerce orders.",
+        "ColumnOrdersDocumentation": "The total number of e-commerce orders which contained this %s at least once.",
+        "ColumnPurchasedProductsDocumentation": "The number of purchased products is the sum product quantity of all e-commerce orders sold.",
         "ColumnQuantityDocumentation": "Quantity is the total number of products sold for each %s.",
         "ColumnRevenueDocumentation": "The total revenue generated by %s.",
         "ColumnRevenuePerVisitDocumentation": "The total revenue generated by %s divided by the number of visits.",
-        "ColumnRevenuePerEntryDocumentation": "The total revenue generated by %s divided by the number of entrances.",
+        "ColumnRevenuePerEntryDocumentation": "The total revenue generated by %s divided by the number of entries.",
```

```suggestion
        "ColumnRevenuePerEntryDocumentation": "The total revenue generated by %s divided by the number of customer entries.",
```
To hopefully avoid confusion with entry pages.